cpp/ql/lib/semmle/code/cpp/ir/implementation/raw/internal/TranslatedElement.qll

```diff
@@ -949,3 +952,11 @@ abstract class TranslatedElement extends TTranslatedElement {
    */
   final TranslatedElement getParent() { result.getAChild() = this }
 }
+
+abstract class TranslatedInstructionContainer extends TranslatedElement {
```

Comment:
This could use at least a one-line QLDoc comment

Reply:
Done